src/meta/src/rpc/service/README.md

```diff
@@ -0,0 +1,87 @@
+# How to write a service
+
+Quick example on how to write a service. The example leader service is implemented [in this PR](https://github.com/risingwavelabs/risingwave/pull/6466)
```

Comment:
How about adding a development folder under `src/docs` and move it there and give it a name like `how-to-write-a-rpc-service.md`? And we'd better using another example like `Health Check Service`, because the `LeaderService` is not representative and #6466 will be closed eventually.

Reply:
Sure. I will use the `Health Service`. I originally used the `leader service` because I am most familiar with it and because we will merge it eventually even if it will not be with https://github.com/risingwavelabs/risingwave/pull/6466